app/controllers/notifications.js

```diff
@@ -27,23 +27,29 @@ export default Ember.Controller.extend({
     return notification;
   },
 
-   setRoute: function(notification) {
-      notification.route = ['orders.detail', notification.order_id];
-   },
+  setRoute: function(notification) {
+    if (notification.category === "message") {
+      return (notification.route = [
```

Comment:
not a huge fan of returning an assignment personally

Reply:
We'll have to return from this block if the condition passes. What i can do is, I can return nothing after assigning the `notification.route`. Is that fine @patrixr ?
Maybe something like 
```if (true) {
a = xyz();
return;
}```